realm_reference_docs_help.md

```diff
@@ -0,0 +1,36 @@
+# Realm Docs Team SDK Reference Manual (API Docs) Collaboration
+
+# Motivation
+
+- API documentation could use some love/attention
```

Comment:
Could you elaborate on whether this means they contain inaccurate information, have grammatical issues, are visually unappealing, etc.?